main.tf

```
@@ -9,65 +9,43 @@ terraform {
       version = ">= 3.26.0"
     }
   }
-  required_version = "~> 0.14"
+
+  required_version = "~> 0.15"
+
   backend "remote" {
-    organization = "<YOUR_TERRAFORM_ORG>"
+    organization = "hashicorp-learn"
```

update to match the tutorial
```suggestion
    organization = "<YOUR_TERRAFORM_ORG>"
```